src/layers/legacy/medLog/CMakeLists.txt

```diff
@@ -28,9 +28,8 @@ find_package(ITK REQUIRED COMPONENTS ITKCommon)
 include(${ITK_USE_FILE})
 
 # find vtk
-find_package(VTK REQUIRED COMPONENTS vtkCommonCore)
-include(${VTK_USE_FILE})
-
+find_package(VTK REQUIRED COMPONENTS CommonCore)
+include_directories(${VTK_INCLUDE_DIRS})
```

Comment:
This line (32) shouldn't be required (applies to other files as well). Normally the fact that you include a VTK module in your target links is enough. At least it worked for me when translating anima to VTK 9

Reply:
Thanks for the tip, i'll test that :)